presto-spi/src/main/java/com/facebook/presto/spi/Connector.java

```diff
@@ -15,6 +15,7 @@
 
 import com.facebook.presto.spi.security.ConnectorAccessControl;
 import com.facebook.presto.spi.session.PropertyMetadata;
+import com.facebook.presto.spi.transaction.IsolationLevel;
 
 import java.util.List;
 import java.util.Set;
```

Comment:
Connector will still be sticking around for connectors that don't want to support multi-statement transactions.